### ISSUE TITLE
Translate alt text

### ISSUE THREAD

**Issue by rileymacisaac** [storyblok/storyblok#785]
**The issue occurs while working with:** (check one with "x")   
- [x] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other:* 

One could argue that this is not a bug, but because alt text is content that is served to the end user and is required for accessible images (excluding decorative images) I believe this should be considered a bug.

**Current behavior:**
If you use the default asset or multi-assets field(s), you can upload an image and provide alt text to that image via the asset manager. However, when translating an asset field there is no way to change the alt text of the image. You could use the description field conditionally check for it in your code, but that is hacky and doesn't directly associate that text with the image itself.

I can get around this with the **advanced-image plugin**, but that can be frustrating any time I'd like to use the multi-assets field as it means I have to make multiple fields and restricted functionality.

**Expected behavior:**
I'd expect to switch my language to french (or whatever other language), check the "translate" checkbox and be able to edit the field, including the alt text, based on my preferences.

Or alternatively, I'd like to go to that asset/image in the asset manager and be able to toggle the language there which would allow me to translate the alt text.

**Steps to reproduce:**
- Navigate to a story with an asset field that is translatable
- Switch your language on that story to anything other than the default language
- Click the "translate" checkbox for the asset field
- Add the same image used in the default language
- Click the image title and look at the `alt text` field, which will be in your default language
- Write in your translated alt text and save the asset/image
- Realize that now your image alt text is now globally using the alt text from your translated language

**Comment by hfrid** [comment]
It's basic SEO to use the primary keyword in the alt-text of an image and not being able to get it translated is a bummer. Please consider adding this as a native feature!

**Comment by alvarosabu** [comment]
Hi @rileymacisaac, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
